build/NuGetPackages/Microsoft.Build.Framework.nuspec

```diff
@@ -15,6 +15,7 @@
     <tags>MSBuild</tags>
     <dependencies>
       <group targetFramework=".NETStandard1.3">
+        <dependency id="System.Resources.ResourceManager" version="4.0.1" />
```

Comment:
It keeps popping up when I do a build. Don't know why. Figured might as well leave it there :(